client/my-sites/earn/home.tsx

```diff
@@ -115,9 +119,24 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		const learnMoreLink = hasSimplePayments
 			? null
 			: { url: supportLink, onClick: () => trackLearnLink( 'simple-payments' ) };
-		return {
-			title: translate( 'Collect one-time payments' ),
-			body: hasSimplePayments
+		let title, body;
+		if ( isEnabled( 'earn/rename-payment-blocks' ) ) {
+			title = translate( 'Collect PayPal payments' );
+			body = hasSimplePayments
+				? translate(
+						'Accept credit card payments via PayPal for physical products, digital goods, services, donations, or support of your creative work.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Accept credit card payments for physical products, digital goods, services, donations, or support of your creative work.' )` *ES Score: 9*